openmdao/components/exec_comp.py

```diff
@@ -424,7 +436,10 @@ def _setup_expressions(self):
                     dct = self.add_input(var, val, **meta)
 
             if var not in init_vals:
-                init_vals[var] = dct['value']
+                if 'value' in dct:
+                    dct['val'] = dct.pop('value')
```

Comment:
not clear that this is needed?  `value` has been replaced by `val` in `kwargs2`/`meta` right?